advisories/hackage/spacecookie/HSEC-2025-0004.md

```diff
@@ -0,0 +1,35 @@
+```toml
+
+[advisory]
+id = "HSEC-2025-0004"
+cwe = []
+keywords = ["gopher"]
+
+aliases = []
+related = []
+
+[[affected]]
+package = "spacecookie"
+cvss = "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:N/A:N/E:U/RL:O/RC:C"
```

Comment:
As mentioned in my other comment, we need to remove the temporal CVSS fields (which our library does not yet support).